package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -346,13 +598,22 @@ define([
     },
 
     /**
-     * Initializes the visualization.
+     * Called when an update operation has been performed with success.
+     *
+     * The [isUpdating]{@link pentaho.visual.base.View#isUpdating} property will now have value `false`.
+     * The [isDirty]{@link pentaho.visual.base.View#isDirty} property will also have value `false`.
```

Comment:
The [isDirty]{@link pentaho.visual.base.View#isDirty} property will also have the value `false`.